changes made

dbaskak opened the pull request.

edanilovets commented on lesson_04/homework_04.py:
```diff
@@ -0,0 +1,87 @@
+# task 05
+""" Виведіть, скільки слів у тексті починається з Великої літери?
+"""
+upper_case_letters = sum(1 for letter in adwentures_of_tom_sawer if letter.isupper())
+print(upper_case_letters)
```
The task is about to check what number of WORDS start with the uppercase letter. This solution is working for current task (but it is not checking only words first letter), but will not work for words like "iPhone, eBay"...

dbaskak replied:
ok

edanilovets commented on lesson_04/homework_04.py:
```diff
@@ -0,0 +1,87 @@
+"""
+print(adwentures_of_tom_sawer_sentences[3].lower())
+
+# task 09
+""" Перевірте чи починається якесь речення з "By the time".
+"""
+start_with = "By the time"
+position = adwentures_of_tom_sawer.find(start_with)
+print(f"Sentence 'By the time' starts at {position} position")
```
The task 09 is asking to output True or False, not the index

edanilovets commented on lesson_04/homework_04.py:
```diff
@@ -0,0 +1,87 @@
+position = adwentures_of_tom_sawer.find(start_with)
+print(f"Sentence 'By the time' starts at {position} position")
+
+# task 10
+""" Виведіть кількість слів останнього речення з adwentures_of_tom_sawer_sentences.
+"""
+count = len(adwentures_of_tom_sawer_sentences[-1].split())
+# print(adwentures_of_tom_sawer_sentences[1])
```
last sentence should contains letters.
 and len should not be 0.
 Try another split() param in task 07

dbaskak replied:
ok

dbaskak commented:
пт, 30 серп. 2024 р. о 14:43 Eugene Danilovets ***@***.***>
пише:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In lesson_04/homework_04.py
> <https://github.com/dbaskak/aqa_01/pull/3#discussion_r1738476331>:
>
> > +upper_case_letters = sum(1 for letter in adwentures_of_tom_sawer if letter.isupper())
> +print(upper_case_letters)
>
> The task is about to check what number of WORDS start with the uppercase
> letter. This solution is working for current task (but it is not checking
> only words first letter), but will not work for words like "iPhone, eBay"...
>
Corrected.

> ------------------------------
>
> In lesson_04/homework_04.py
> <https://github.com/dbaskak/aqa_01/pull/3#discussion_r1738477695>:
>
> > +# task 09
> +""" Перевірте чи починається якесь речення з "By the time".
> +"""
> +start_with = "By the time"
> +position = adwentures_of_tom_sawer.find(start_with)
> +print(f"Sentence 'By the time' starts at {position} position")
>
> The task 09 is asking to output True or False, not the index
>
Corrected.

> ------------------------------
>
> In lesson_04/homework_04.py
> <https://github.com/dbaskak/aqa_01/pull/3#discussion_r1738479104>:
>
> > +# task 10
> +""" Виведіть кількість слів останнього речення з adwentures_of_tom_sawer_sentences.
> +"""
> +count = len(adwentures_of_tom_sawer_sentences[-1].split())
> +# print(adwentures_of_tom_sawer_sentences[1])
>
> last sentence should contains letters.
> and len should not be 0.
> Try another split() param in task 07
>
Corrected
Thanks for you time

> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dbaskak/aqa_01/pull/3#pullrequestreview-2272031670>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCK2667HADRAZ3GQ6SHSFDZUBLHXAVCNFSM6AAAAABNK62KYKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENZSGAZTCNRXGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

edanilovets commented:
@dbaskak When you updated PR , please click Resolve Conversation buttons. And it is a good practice to answer directly on each conversation
<img width="1330" alt="Screenshot 2024-08-31 at 15 57 50" src="https://github.com/user-attachments/assets/d35fc8bd-803f-4380-b68f-0eaad8924bc5">

dbaskak commented:
Ok, will keep it in mind for further

On Sat, 31 Aug 2024 at 15:58, Eugene Danilovets ***@***.***>
wrote:

> @dbaskak <https://github.com/dbaskak> When you updated PR , please click
> Resolve Conversation buttons. And it is a good practice to answer directly
> on each conversation
> Screenshot.2024-08-31.at.15.57.50.png (view on web)
> <https://github.com/user-attachments/assets/d35fc8bd-803f-4380-b68f-0eaad8924bc5>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dbaskak/aqa_01/pull/3#issuecomment-2322889987>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCK265NQ6BVYB6RI425BBTZUG4XNAVCNFSM6AAAAABNK62KYKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMRSHA4DSOJYG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
